### PR TITLE
Update doc for Preference widget and advance configuration

### PR DESCRIPTION
**Pull request by DISW-Mendix-3DViewer**
Doc update for 3D Viewer

Update the doc of widget-*Preference* and Advance configuration

### CODE REVIEW

**Comment by DISW-Mendix-3DViewer** [comment]
HI @MarkvanMents , pls help review the MR. Thanks!

File: content/en/docs/partners/siemens/3d-viewer/_index.md [hunk 1]
```diff
@@ -777,6 +781,84 @@ This defines measurement behaviors and dimension styles. The `measurement` secti
...
+    "PART_SPECIFICATION",
+    "PMI_TABLE_TYPE",
+    "PROCESS_SPECIFICATION",
+    "REFERENCE_AXIS",
+    "REFERENCE_GEOMETRY",
+    "REFERENCE_PLANE",
+    "REFERENCE_POINT",
+    "SECTION",
+    "SPOT_WELD",
+    "SURFACE_FINISH",
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/3d-viewer/_index.md]: This looks like invalid JSON - shouldn't have a comma after the last item in an array?

File: content/en/docs/partners/siemens/3d-viewer/_index.md [hunk 2]
```diff
@@ -589,11 +587,11 @@ The **Usage Dashboard** shows the real-time statistics about the usage of an app
 
 ### 5.9 Setting Preferences {#set-preferences}
 
-You can change the setting of the Preference widget to customize the behavior of the model and setup to your requirements.
+You can change the settings of the Preferences widget to customize the behavior of the model and setup to suit your needs.
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/3d-viewer/_index.md]: Is the widget called "Preference" or "Preferences"? The image below says "Preference". 
image
```suggestion
You can change the settings of the Preference widget to customize the behavior of the model and set it up to suit your needs.
```

File: content/en/docs/partners/siemens/3d-viewer/_index.md [hunk 3]
```diff
@@ -589,11 +587,11 @@ The **Usage Dashboard** shows the real-time statistics about the usage of an app
...
 
-{{<figure src="/attachments/partners/siemens/3d-viewer/preferences-general.png" alt="preferences-general" >}}
+{{< figure src="/attachments/partners/siemens/3d-viewer/preferences-general.png" alt="preferences-general" >}}
 
-On the **PMI** tab, you can set filtering options for PMI and click **Ok** to save the changes.
+Click the "Preference" button to display the dialog. You can set your filter as a preference in any combination you like, and when you click the **OK** button, the filter will take effect the next time you open a file; the **Reset** button clears all settings. This method can only set the preference when your application is running, but of course we also provide a more advanced method - before the application starts - to set your preference, for more details you can refer to [Advanced Configuration](#advance_configuration).
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/3d-viewer/_index.md]:
```suggestion
Click **Preference** to display the dialog. You can set your filter as a preference in any combination you like, and when you click the **OK** button, the filter will take effect the next time you open a file; the **Reset** button clears all settings. This method can only set the preference when your application is running, but of course we also provide a more advanced method - before the application starts - to set your preference, for more details you can refer to [Advanced Configuration](#advance_configuration).
```

File: content/en/docs/partners/siemens/3d-viewer/_index.md [hunk 4]
```diff
@@ -777,6 +781,84 @@ This defines measurement behaviors and dimension styles. The `measurement` secti
...
+    }
+}
+```
+
+#### 6.5.1 ignoredPMI
+
+Defines the PMI entities you do not want to display.
+
+* Type – array
+* Default value - *[ ]*
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/3d-viewer/_index.md]: This gets interpreted wrongly by markdown.
```suggestion
* Default value - *`[ ]`*
```

**Review by MarkvanMents — COMMENTED** [review]
Hi @digital-engineering-services-sample-app 
I've made a couple of suggestions for changes. I'll try to commit them through the PR (I can't commit them directly to your branch).

The only thing is that the settings seem to be called "Preference" but sometimes we say it is called "Preferences". I think we should be consistent - perhaps stick to "Preference" since that is what appears in the UX.

Let me know whether this needs to be made consistent, and the best way of doing this. Then I can publish as soon as you are ready.

**Comment by MarkvanMents** [comment]
@digital-engineering-services-sample-app 
I am not able to commit my changes. Can you commit them for me?

**Comment by MarkvanMents** [comment]
Alternatively, give me access to your fork of the repo?

File: content/en/docs/partners/siemens/3d-viewer/_index.md (hunk 1 above)
Comment (DISW-Mendix-3DViewer) [reply content/en/docs/partners/siemens/3d-viewer/_index.md]: Yes, it is a mistake from my side

**Comment by DISW-Mendix-3DViewer** [comment]
Thank you @MarkvanMents very much for the review, very detailed.
I made changes to all the comments.

**Review by MarkvanMents — APPROVED** [review]
Hi @digital-engineering-services-sample-app 

Can this be published immediately?
(I'm actually out of office the next two days but can do this as soon as I come back - but I'll also ask someone to keep an eye on this PR if you want it published before then).

**Comment by DISW-Mendix-3DViewer** [comment]
> Hi @digital-engineering-services-sample-app
> 
> Can this be published immediately? (I'm actually out of office the next two days but can do this as soon as I come back - but I'll also ask someone to keep an eye on this PR if you want it published before then).

Yes, it can be published immediately, it has no other dependencies

**Comment by katarzyna-koltun-mx** [comment]
@digital-engineering-services-sample-app Thank you for confirming! I'll merge the changes immediately.

**Comment by katarzyna-koltun-mx** [comment]
@MarkvanMents @digital-engineering-services-sample-app The changes are merged and should appear in production today.

**Pull request merged**